src/exchanges.js

```diff
@@ -51,6 +54,33 @@ const nbuExchange = async () =>  {
   console.table(data);
 };
 
+const currencyCodeNumber = async () => {
+  const question = 'Enter currency code or its number:\n';
+  while (true) {
+    const request = await promised.question(question);
+
+    if (/^\d+$/.test(request)) {
+      if (codesList[parseInt(request)] === undefined) {
+        console.log(`${red}` +
+          'Something went wrong!\nMake sure you entered correct data.' +
+          `${green}`);
+      } else console.log(`${codesList[parseInt(request)]}`);
```

Comment:
```suggestion
      } else {
        console.log(`${codesList[parseInt(request)]}`);
      }
```
for block consistency